conda/environments/cuxfilter_dev_cuda11.0.yml

```
@@ -39,7 +39,7 @@ dependencies:
 - ipython
 - jupyterlab
 - flake8=3.8.3
-- black=19.10
+- black=23.3.0
```

@AjayThorve I think you meant 22.3.0? CI appears to pass (?), but the version 23.3.0 doesn't exist.
```suggestion
- black=22.3.0
```

---

@bdice, it is indeed a typo, thanks for pointing it out :pray:. I think CI passes as it pulls the env from `integration` repo.

fixed it!

---

Awesome! 😎 Resolving this thread.